examples/chat.rs

```diff
@@ -1,3 +1,11 @@
+//! Example chat application.
+//!
+//! Run with
+//!
+//! ```
+//!     cargo run --features="ws" --example chat
```

Comment:
nitpick:

```suggestion
//! cargo run --features=ws --example chat
```

Reply:
This would also make sense to have for all other example. Doesn't have to be done in this PR though.